test/com/google/javascript/jscomp/ProcessEs6ModulesTest.java

```diff
@@ -108,6 +108,10 @@ public void testImport() {
     testModules(
         "import x, {f as foo, b as bar} from 'other'; use(x);",
         LINE_JOINER.join("goog.require('module$other');", "use(module$other.default);"));
+
+    testModules(
+        "import {default as name} from 'other'; use(name);",
+        LINE_JOINER.join("goog.require('module$other');", "use(module$other.default);"));
```

Comment:
The purpose of the line joiner is to put the strings one on each line so it reads more naturally. Either do that or just put it all on one line in one string.